cmd/ferretdb/main.go

```diff
@@ -57,56 +57,55 @@ import (
 // It's used for parsing the user input.
 //
 // Keep structure and order in sync with documentation and embeddable package.
-// TODO https://github.com/FerretDB/FerretDB/issues/4746
+//
+//nolint:lll // Long struct tags need to be formatted for readability.
 var cli struct {
 	// We hide `run` command to show only `ping` in the help message.
 	Run  struct{} `cmd:"" default:"1"                             hidden:""`
-	Ping struct{} `cmd:"" help:"Ping existing FerretDB instance."`
+	Ping struct{} `cmd:"" help:"Ping existing FerretDB instance." group:"general"`
```

Comment:
Ping is not a flag, we don't need to add `group` for this :hugs:

Reply:
@chilagrow removed ping from group